tests/e2e/.env.example

```
@@ -8,5 +8,11 @@ TEST_PRIVATE_KEY_CONTRACT=YOUR_CONTRACT_PRIVATE_KEY_HERE
 # Secondary private key (used by some document/token transitions)
 TEST_PRIVATE_KEY_SECONDARY=YOUR_TEST_PRIVATE_KEY_SECONDARY
 
+# Platform Address lifecycle. Each address must match its corresponding WIF.
+TEST_PLATFORM_ADDRESS_A=YOUR_FIRST_TESTNET_PLATFORM_ADDRESS
+TEST_PLATFORM_ADDRESS_KEY_A=YOUR_FIRST_TESTNET_ADDRESS_WIF
+TEST_PLATFORM_ADDRESS_B=YOUR_SECOND_TESTNET_PLATFORM_ADDRESS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TEST_PLATFORM_ADDRESS_B key should go before the TEST_PLATFORM_ADDRESS_KEY_A key

(UnorderedKey)